test/GovLst.t.sol

```diff
@@ -5020,3 +5042,35 @@ contract ConvertToRebasingAndUnstake is GovLstTest {
     lst.convertToRebasingAndUnstake(_account, _shares);
   }
 }
+
+contract Delegates is GovLstTest {
+  function testFuzz_Delegates(address _holder, address _delegatee, uint256 _amount) public {
```

Comment:
* Needs a more expressive name for scopelint
* We should also add a new test that demonstrates the return value from this method always equals the return value from `delegateeForHolder`

Reply:
The more expressive name commit: https://github.com/withtally/stGOV/commit/3d7a520307fe8a12df63ece44b2671ceaec41bb5

Reply:
The additional test commit: 8e6ad0e8a2f829a76baf975e4be1a1ed7fdd6dab